sql-server-2014-alwayson-dsc/azuredeploy.json

```diff
@@ -150,12 +128,67 @@
       },
       "defaultValue": "sqlservice"
     },
+
     "sqlServerServiceAccountPassword": {
       "type": "securestring",
       "metadata": {
         "description": "The SQL Server Service account password"
       }
     },
+
+    "VmNetworkSubnet": {
```

Comment:
I will not fix this comment. I would like to keep those as parameters and the values can be changed during deployment